package.json

```diff
@@ -42,6 +42,6 @@
     "super-request": "0.0.8",
     "supertest": "^0.13.0",
     "supertest-as-promised": "^1.0.0",
-    "valid-url": "^1.0.9"
+    "validator": "^3.39.0"
```

Comment:
Instead of this, I think it would be best to use a fully compliant validator such as [isemail](https://www.npmjs.com/package/isemail) this way we are able to better handle edge cases other than the standard `something@somewhere.tld`.